### ISSUE TITLE
Whether the header allows line breaks

### ISSUE THREAD

**Issue by Progentalwf** [glideapps/glide-data-grid#930]
`const columns = [
  {
    title: "AAAA\nBBBB",
    allowWrapping: true,
  },
...
] `

**Comment by A-Posthuman** [comment]
I agree it would be nice if somehow the header text could be wrapped, and in a controllable way.

Also would be good if the headers could take a contentAlign or something to allow the text to be center or right aligned.

**Comment by portare** [comment]
@Progentalwf  Did u find solution?

**Comment by BrianHung** [comment]
For contentAlign, you can implement that yourself using [`drawHeader`](https://docs.grid.glideapps.com/api/dataeditor/custom-cells#drawheader) API. Would love if the default drawHeader did support that field though. Text wrapping is a bit harder, just because headerHeight exists as an API already; I haven't dug into how allowWrapping in TextCells interact with rowHeight since that's a similar behavior.
